app/scripts/model/dataProxy.js

```diff
@@ -703,9 +706,9 @@ window.DataManagerForIviz = (function($, _) {
             });
           $.when.apply($, requests).then(function() {
             _.each(_profiles, function(_profile) {
-              if (_profile.study_id + '_gistic' === _profile.id) {
-                self.gisticProfileIdsMap[_profile.study_id] = _profile.id;
-              } else if (_profile.study_id + '_mutations' === _profile.id) {
+              if (_profile.genetic_alteration_type === 'COPY_NUMBER_ALTERATION') {
```

Comment:
Once the issue is fixed(https://github.com/cBioPortal/cbioportal/issues/1645), I'll update this.

It should be like 
if(_profile.genetic_alteration_type === 'COPY_NUMBER_ALTERATION' && _profile.show_profile_in_analysis_tab){....}